docs/lib/QUICKSTART.md

```diff
@@ -422,26 +424,12 @@ When the uVisor is enabled, all NVIC APIs are rerouted to the corresponding uVis
 
 - The uVisor owns the interrupt vector table.
 - All ISRs are relocated to SRAM.
-- Code in a box can only change the state of an IRQ (enable it, change its priority, etc.) if the box registered that IRQ with uVisor at runtime, using the `NVIC_SetVector` API.
+- Code in a box can only change the state of an IRQ (enable it, change its priority, etc.) if the box registered that IRQ with uVisor via an IRQ ACL.
 - An IRQ that belongs to a box can only be modified when that box context is active.
 
-Although this behavior is different from that of the original NVIC, it is backward compatible. Legacy code (such as a device HAL) still works after uVisor is enabled. The general use case is the following:
-
-```C
-#define MY_IRQ 42
-
-/* Set the ISR for MY_IRQ at runtime.
- * Without uVisor: Relocate the interrupt vector table to SRAM and set my_isr as
-                   the ISR for MY_IRQ.
- * With    uVisor: Register MY_IRQ for the current box with my_isr as ISR. */
-NVIC_SetVector(MY_IRQ, &my_isr);
-
-/* Change the IRQ state. */
-NVIC_SetPriority(MY_IRQ, 3);
-NVIC_EnableIRQ(MY_IRQ);
-```
+Although this behavior is different from that of the original NVIC, it is backward compatible. Legacy code (such as a device HAL) still works after uVisor is enabled.
 
-> **Note**: In this model, a call to `NVIC_SetVector` must happen before an IRQ state changes. In platforms that don't relocate the interrupt vector table, such a call might be absent and must be added to work with uVisor.
+An IRQ that has not been listed in the ACL list of any box will be implicitly claimed by the first box that accesses it via an NVIC API.
```

Comment:
If it won't change the meaning, please change this sentence to "The first box that accesses an IRQ not in the ACL list of any box implicitly claims the IRQ through an NVIC API."